docs/developer-docs/docs/operate-a-node/node-commands.md

```diff
@@ -0,0 +1,145 @@
+﻿---
+sidebar_position: 7
+---
+
+# Node commands
+
+## Overview
+
+**Node commands** are commands for interacting with a node through the command line. In this article, you'll find a brief overview of commands for interacting with the Warden node, `wardend`.
+
+These commands allow you to do the following:
+
+- Query the node
+- Initiate transactions
+- Manage your key
+- Edit the genesis file
+- and much more
+
+## Execute `wardend` commands
+
```

Comment:
This statement is misleading. You can use `wardend` commands if the binary is installed, irrespective if the chain is running or not.